components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -1677,6 +1677,15 @@ private void setRuntimeArtifactsSyncGatewayConfig (OMElement omElement){
             log.debug("Retry Duration Element is not set. Set to default duaration");
         }
 
+        OMElement startupConfigElement = omElement.getFirstChildWithName(
```

Comment:
startupConfigElement -> Retrieval Mode ?
@pubudu538 ?

Reply:
+1

Reply:
Data Retrieval Mode is meaningful.

Reply:
https://github.com/wso2/carbon-apimgt/pull/8945/commits/dfeccf937e8c48b78ccfe0de83e8e04686c1b1f1